.github/workflows/performance-benchmarking.yml

```
@@ -0,0 +1,596 @@
+            github.rest.issues.createComment({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: `## 🚀 Performance Benchmark Results\n\n${report}\n\n*This comment was automatically generated by the performance benchmarking workflow.*`
+            });
+
+  benchmark-summary:
+    name: Benchmark Summary
+    runs-on: ubuntu-latest
+    needs: [baseline-benchmarks, current-benchmarks, phase3-benchmarks, performance-comparison]
+    if: always() && needs.should-run-benchmarks.outputs.run_benchmarks == 'true'
```

actionlint (1.7.7)

property "should-run-benchmarks" is not defined in object type {baseline-benchmarks: {outputs: {baseline_results: string}; result: string}; current-benchmarks: {outputs: {current_results: string}; result: string}; performance-comparison: {outputs: {}; result: string}; phase3-benchmarks: {outputs: {}; result: string}}

(expression)